src/smpp_gateway/management/commands/listen_mo_messages.py

```diff
@@ -4,7 +4,7 @@
 
 
 class Command(BaseCommand):
-    help = "Listen for MO messages."
+    help = "Receive inbound MO messages and pass them to RapidSMS."
```

Comment:
I'm hopefully leaving more helpful hints for people new to SMPP and RapidSMS like me.